include/ur_modern_driver/ros/mb_publisher.h

```diff
@@ -28,9 +30,12 @@ class MBPublisher : public URStatePacketConsumer
   }
 
   void publish(ur_msgs::IOStates& io_msg, SharedMasterBoardData& data);
+  void publishRobotStatus(const RobotModeData_V1_X& data) const;
+  void publishRobotStatus(const RobotModeData_V3_0__1& data) const;
 
 public:
   MBPublisher() : io_pub_(nh_.advertise<ur_msgs::IOStates>("ur_driver/io_states", 1))
+                , status_pub_(nh_.advertise<industrial_msgs::RobotStatus>("robot_status", 1))
```

Comment:
Should this perhaps be published at "ur_driver/robot_status"?

Reply:
We could, but that would make it non-compliant with other ros-i robot drivers.

A remap can fix that though.

Reply:
Alright yes that makes perfect sense, let's keep it as it is here then.

Reply:
Well .. let's actually not.

All topics for the `ur_driver` are below a `ur_driver` namespace already, so having this one outside it breaks that.

If/when we need to make this driver ros-i compliant, we can either blanketly remap everything or update the topics.

Reply:
So I'll add the prefix for consistencies sake?

Reply:
I believe that makes the most sense for now, yes.

Otherwise users would find all topics of the driver under the `ur_driver` namespace, but a lone (and seemingly unrelated) `/robot_status` topic in the global one.